doc/sphinx/user/methods/melt-transport.md

```diff
@@ -176,3 +176,24 @@ so we get
 More details on the implementation can be found in {cite:t}`dannberg:heister:2016`.
 A benchmark case demonstrating the propagation of solitary waves can be
 found in {ref}`sec:benchmarks:solitary_wave`.
+
+(sec:methods:darcy-flow)=
+# Calculations with Darcy flow
+To calculate fluid transport, ASPECT can advect compositional fields with the fluid velocity
+computed using Darcy's Law (derived in McKenzie 1984). Currently, this method approximates the flud
+pressure gradient as the lithostatic pressure, so the expression used for the fluid velocity is:
+
+```{math}
+\bold{u_f} = \bold{u_s} - \frac{K_D}{\phi} \left(\rho_s - \rho_f \right)\bold{g}
```

Comment:
You need to use `\mathbf{u}_f` instead of bold, otherwise this is not rendered correctly.